lib/es-models/bib.js

```diff
@@ -226,14 +227,15 @@ class EsBib extends EsBase {
   dates () {
     const dates = []
     const rawMarc = this.bib.varField('008')
+    const id = this.bib.id
```

Comment:
Nit: Given changes suggested below, if this alias has very little or singular use, maybe lets not create it?